sapi/fpm/fpm/fpm_conf.c

```diff
@@ -285,7 +285,7 @@ static char *fpm_conf_set_long(zval *value, void **config, intptr_t offset) /* {
 static char *fpm_conf_set_time(zval *value, void **config, intptr_t offset) /* {{{ */
 {
 	char *val = Z_STRVAL_P(value);
-	int len = strlen(val);
+	int len = (int) strlen(val);
```

Comment:
Shouldn't Z_STRLEN_P(val) give the value here instead of re-counting it?

Reply:
There is a difference, if the PHP string contains NUL bytes. Not sure, if that might be the case here.